atlasdb-dropwizard-bundle/versions.lock

```diff
@@ -763,35 +763,36 @@
             ]
         },
         "org.eclipse.jetty:jetty-continuation": {
-            "locked": "9.2.17.v20160517",
+            "locked": "9.4.0.v20161208",
```

Comment:
We are using the jetty-alpn-agent(https://github.com/jetty-project/jetty-alpn-agent) instead so that the boot jar is loaded as per the JVM version.